src/redux/books/books.js

```diff
@@ -0,0 +1,20 @@
+const ADDBOOK = {
+  type: 'ADD',
+};
+
+const REMOVEBOOK = {
+  type: 'REMOVEBOOK',
+};
+
```

Comment:
- [ ] I noticed that you didn't create actions for adding and removing books in the book slice file. Kindly create actions using the [duck](https://github.com/erikras/ducks-modular-redux) pattern. You can also refer to point 1 in my comment ...